### PR TITLE
focus the editor hover on escape of accessible view so focus is not lost

### PR DESCRIPTION
**Pull request by meganrogge**
fix part of #188822

### CODE REVIEW

File: src/vs/workbench/services/hover/browser/hoverService.ts [hunk 1]
```diff
@@ -145,6 +145,8 @@ function getHoverOptionsIdentity(options: IHoverOptions | undefined): IHoverOpti
 
 class HoverContextViewDelegate implements IDelegate {
 
+	container: HTMLElement | undefined;
```
Comment (roblourens) [inline src/vs/workbench/services/hover/browser/hoverService.ts:148]: Unused?

File: src/vs/workbench/services/hover/browser/hoverService.ts [hunk 2]
```diff
@@ -156,6 +158,7 @@ class HoverContextViewDelegate implements IDelegate {
 	}
 
 	render(container: HTMLElement) {
+		this.container = container;
```
Comment (meganrogge) [inline src/vs/workbench/services/hover/browser/hoverService.ts:161]:
```suggestion
```

File: src/vs/workbench/services/hover/browser/hoverService.ts [hunk 3]
```diff
@@ -145,6 +145,8 @@ function getHoverOptionsIdentity(options: IHoverOptions | undefined): IHoverOpti
 
 class HoverContextViewDelegate implements IDelegate {
 
+	container: HTMLElement | undefined;
+
```
Comment (meganrogge) [inline src/vs/workbench/services/hover/browser/hoverService.ts:149]:
```suggestion
```

File: src/vs/workbench/services/hover/browser/hoverService.ts [hunk 4]
```diff
@@ -145,6 +145,7 @@ function getHoverOptionsIdentity(options: IHoverOptions | undefined): IHoverOpti
 
 class HoverContextViewDelegate implements IDelegate {
 
+	container: HTMLElement | undefined;
```
Comment (meganrogge) [inline src/vs/workbench/services/hover/browser/hoverService.ts:148]:
```suggestion
```

**Review by roblourens — APPROVED** [review]

**Pull request merged**